app/test_main.py

```diff
@@ -2,0 +1,40 @@
+from app.main import can_access_google_page
+from unittest import TestCase
+from unittest.mock import patch
+
+
+@patch("app.main.has_internet_connection")
+@patch("app.main.valid_google_url")
+class TestCanAccessGoogle(TestCase):
+
+    def test_return_accessible_if_both_true(
+            self, mock_validator: patch,
+            mock_connection: patch
+    ) -> None:
+        mock_validator.return_value = True
+        mock_connection.return_value = True
+        assert can_access_google_page("test.url") == "Accessible"
+
+    def test_return_not_accessible_if_url_not_valid(
+            self, mock_validator: patch,
+            mock_connection: patch
+    ) -> None:
+        mock_validator.return_value = False
+        mock_connection.return_value = True
+        assert can_access_google_page("test.url") == "Not accessible"
+
+    def test_return_not_accessible_if_no_internet(
```

Comment:
The test method name `test_return_not_accessible_if_no_internet` could be more descriptive. According to the checklist, use descriptive and correct test names. Consider a name like `test_valid_url_no_connection`.